vocabularies/borehole-design.ttl

```diff
@@ -0,0 +1,51 @@
+@prefix bhd: <http://linked.data.gov.au/def/borehole-design/> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/borehole-design> a owl:Ontology .
+
+bhd:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-10-03T16:24:51+10:00"^^xsd:dateTime ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:modified "2019-10-03T16:32:36+10:00"^^xsd:dateTime ;
+    dct:source "Developed by the Geological Survey of Queensland." ;
+    skos:definition "A description of the intended configuration and trajectory of a borehole"@en ;
+    skos:hasTopConcept bhd:deviated,
+        bhd:inclined,
+        bhd:lateral,
+        bhd:vertical ;
+    skos:prefLabel "Borehole Inclination Design"@en .

```

Comment:
Does this need to be consistent with the name of the file?
_"borehole-inclination-design.ttl"_ or _prefLabel is "Borehole Design"_

Reply:
Changed. Forgot to change the prefLabel when i changed the file name and URIs.